resource-service/mvn_dependency_tree.txt

```diff
@@ -62,7 +62,7 @@ com.google.inject/guice/version = 4.1.0
 
 com.google.inject.extensions/guice-servlet/version = 4.0
 
-com.google.protobuf/protobuf-java/version = 3.5.1
+com.google.protobuf/protobuf-java/version = 2.5.0
```

Comment:
Downgraded dependency.

Reply:
Dependency tree shows:
protobuf at 3.5.1 for spring-boot-starter-parent but
for hadoop-common the version is +- org.apache.hadoop:hadoop-common:jar:3.2.0:compile ==> 2.5.0
We cannot fix the version used by Hadoop.

Reply:
The data service should no longer need the Hadoop dependency once the Hadoop reader has been moved from it to run separately. Is the data reader still a part of the data service at the moment? If so, we can leave this and it will disappear once the separation work is carried out.

Reply:
Hi @m09526 - the resourceService contains the HadoopResourceService which makes use of Configuration for instance which is part of the hadoop-common library. Does this seem incorrect???